src/lib.rs

```diff
@@ -138,6 +139,7 @@ pub use property_attribute::*;
 pub use property_descriptor::*;
 pub use property_filter::*;
 pub use property_handler_flags::*;
+pub use regexp::RegExpCreationFlags;
```

Comment:
```suggestion
pub use regexp::RegExpCreationFlags;
pub use regexp::Regexp;
```

` regexp::Regexp` needs to be public too?

Reply:
`RegExp` is public already as it is defined in `data.rs`